.github/workflows/actions.yml

```diff
@@ -0,0 +1,34 @@
+name: main
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  main-job:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
```

Comment:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

This is set to true by default, we can revisit ZAF and other places where we may have these. Its not a big deal to just leave them there, but for this one we can remove it, since its not merged yet.